endpoint.h

```diff
@@ -22,6 +22,9 @@
 #include "comm.h"
 #include "pollable.h"
 
+#define READ_MESSAGE_OK 1
+#define READ_MESSAGE_UNIDENTIFIED 2
```

Comment:
Do you mean just these two? Or all of the `read_msg` returns? Because currently, it may also return a whole range of 'invalid' values, starting on 0 to any negative `errno`.